src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationServiceTest.kt

```diff
@@ -229,6 +233,48 @@ internal class RecommendationServiceTest : ServiceTestBase() {
     }
   }
 
+  @Test
+  fun `create recommendation when one in progress already exists for case`() {
```

Comment:
Is the test name correct here? Shouldn't it be something like "return existing recommendation when one in progress already exists for case and create endpoint hit" or similar